storybook/stories/LeftToRightRowAnimation/index.tsx

```diff
@@ -7,14 +7,13 @@
  * @lint-ignore-every XPLATJSCOPYRIGHT1
  */
 
-import React, { Component, RefObject, Ref } from "react";
+import React, { Component, RefObject } from "react";
 import { View, FlatList, SafeAreaView } from "react-native";
 import Button from "../../../components/Button";
-import RowView from "../../../components/AnimatableRowView";
+import RowView from "../../../components/SlidingRowView";
```

Comment:
This file is exactly the same as [TopToBottomRowAnimation](https://github.com/juanjo-ramos/react-native-workshop/blob/master/storybook/stories/TopToBottomRowAnimation/index.tsx) but this line.

Could we create an interface that both row classes conform to?

Reply:
I think we could, but I think that may distract a bit the purpose of the workshop if we have to explain it.

Reply:
I didn't mean to explain that as part of the workshop. I was more thinking for our benefit. As in leaving the codebase the deal with in a cleaner state. They won't need to work on that file at all.

Reply:
Merging as is, we can think about this later